docs/_source/guides/llms/practical_guides/fine_tune_others.md

```diff
@@ -0,0 +1,194 @@
+
+#### Multi-label
+Multi-label text classification is generally more complex than single-label classification due to the challenge of determining and predicting multiple relevant labels for each text. It finds applications in various domains, including document tagging, topic labeling, and content recommendation systems.
+
+### Training
+
+Data for the training text classification using our `FeedbackDataset` is defined by following three easy steps.
+
+1. We need to define unification strategies a `RatingUnification`, a `LabelUnification` or a `MultiLabelUnification`.
+
+2.  For this task, we assume we need a `text-label`-pair for defining a text classification task. We allow mapping for creating a `TrainingTaskMapingForTextClassification` by mapping `*Field` to a `text`-value and allow for mapping a `RatingUnification`, `LabelUnification` or a `MultiLabelUnification` to a `label`-value.
+
+3.  We then define an `ArgillaTrainer` instance with support for "openai", "setfit", "peft", "spacy" and "transformers".
+
+#### Unify responses
+
+Argilla `*Question`s need to be [unified using a strategy](/guides/llms/practical_guides/collect_responses) and so do `RatingQuestions`s, `LabelQuestion`s and `MultiLabelQuestion`s. Therefore, we first need to define a `*QuestionUnification`, which takes one of the questions and one of their associated strategies.
+
+````{note}
+Note that `RatingQuestion`s can be unified using a "majority"-, "min"-, "max"- or "disagreement"-strategy. Both `LabelQuestion`s and `MultiLabelQuestion`s can be resolved using a "majority"-, or "disagreement"-strategy.
+````
+
+::::{tab-set}
+
+:::{tab-item} RatingQuestion
+```python
+from argilla import RatingQuestion, RatingUnification
+
+label_unification = RatingUnification(
+    question=LabelQuestion(...),
+    strategy="majority" # or "min", "max", "disagreement"
+)
+```
+:::
+
+:::{tab-item} LabelQuestion
+```python
+from argilla import LabelQuestion, LabelUnification
+
+label_unification = LabelUnification(
+    question=LabelQuestion(...),
+    strategy="majority" # or "disagreement"
+)
+```
+:::
+
+:::{tab-item} MultiLabelQuestion
+```python
+from argilla import MultiLabelQuestion, MultiLabelUnification
+
+label_unification = MultiLabelUnification(
+    question=MultiLabelQuestion(...),
+    strategy="majority" # or "disagreement"
+)
+```
+:::
+
+::::
+
+#### Define a task mapping
+
+After defining the `unfication`, we can now define our `TrainingTaskMapingForTextClassification`.
+
+```python
+from argilla import FeedbackDataset, TrainingTaskMapingForTextClassification
+
+label_unification = ...
+dataset = FeedbackDataset(...)
```

Comment:
Same thing here, one cannot just init a `FeedbackDataset` and expect it to have records, so I'd suggest to showcase either `from_argilla` or `from_huggingface`